.env.test

```diff
@@ -0,0 +1,11 @@
+# Flask server configuration.
+FLASK_APP=app
+FLASK_ENV=development
+FLASK_SKIP_DOTENV=True
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+API_KEY=api123
+TOKEN=token123
+BOARD_ID=board123
+TODO_LIST_ID=456
```

Comment:
For completeness I'd include DONE_LIST_ID even though your tests don't use it yet. Also in your e2e test setup